src/templates/list-table-block/with-fetch-ordering-and-pagination.vue

```diff
@@ -431,6 +431,7 @@
   const isRenderOneOption = props.actions?.length === 1
   const selectedId = ref(null)
   const dataTableRef = ref(null)
+  const paginationFirstElement = ref(0)
```

Comment:
Não poderíamos utilizar o `firstItemIndex.value` 
```suggestion
  const paginationFirstElement = ref(0)
```